test/kickstart-templates/kickstart.ks.template

```diff
@@ -28,6 +28,7 @@ part /boot --fstype=xfs --asprimary --size=800
 part pv.01 --grow
 volgroup rhel pv.01
 logvol / --vgname=rhel --fstype=xfs --size=REPLACE_LVM_SYSROOT_SIZE --name=root
+logvol none --thinpool --size=REPLACE_LVM_THINPOOL_SIZE --fstype=None --name=ushift-thin --vgname=rhel
```

Comment:
This code can also be added in the suite setup and cleaned in the teardown. Something like:

```
Config LVM Pool
    [Documentation]    Create LVM Thin Pool
    ${stdout}    ${rc}=    Execute Command
    ...    lvcreate --type thin-pool --thinpool ushift-thin --size 1024 rhel
    ...    sudo=True    return_rc=True
    Should Be Equal As Integers    0    ${rc}

Delete LVM Pool
    [Documentation]    Delete LVM Thin Pool
    ${stdout}    ${rc}=    Execute Command
    ...    lvremove -f -y lvremove rhel/ushift-thin
    ...    sudo=True    return_rc=True
    Should Be Equal As Integers    0    ${rc}
```

Reply:
+1 I'll extract thinpool handling from the kickstart and handled it in the test suite.

cc @dhellmann